content/admin/configuration/configuring-your-enterprise/configuring-rate-limits.md

```diff
@@ -61,3 +61,44 @@ You can apply Git rate limits per repository network or per user ID. Git rate li
 3. Type limits for each repository network or user ID.
   ![Fields for repository network and user ID limits](/assets/images/enterprise/management-console/example-git-rate-limits.png)
 {% data reusables.enterprise_management_console.save-settings %}
+
+{% ifversion ghes > 3.4 %}
+
+## Configuring rate limits for {% data variables.product.prodname_actions %}
+
+You can apply a rate limit to {% data variables.product.prodname_actions %} workflow runs. For more information about {% data variables.product.prodname_actions %}, see "[About {% data variables.product.prodname_actions %} for enterprises](/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/about-github-actions-for-enterprises)."
+
+### About rate limits for {% data variables.product.prodname_actions %}
+
+Your {% data variables.product.product_name %} instance assigns each {% data variables.product.prodname_actions %} workflow job to a runner. If your instance cannot immediately assign a job to an available runner, the job will wait in a queue until a runner is available. If {% data variables.product.prodname_actions %} experiences sustained high load, the queue can back up, and the performance of {% data variables.product.product_location %} may degrade.
+
+To avoid this performance degradation, you can configure a rate limit for {% data variables.product.prodname_actions %}. This rate limit is expressed in job runs per minute. {% data variables.product.product_name %} calculates and applies the rate limit for the sum total of all job runs on the instance. If runs exceed the rate limit, additional runs will fail instead of entering the queue.
+
+![Screenshot of a rate-limited workflow run](/assets/images/enterprise/actions/actions-rate-limited.png)
```

Comment:
Maybe just a text representation of the error would be more useful than this screenshot (the error text is really small in the image), and it would also have the added benefit of being searchable both for SEO and our site search.